### PR TITLE
fix: inherit notifiers.smtp from root smtp configuration

### PR DESCRIPTION
**Pull request by kamiiiel**
## Issue

https://gravitee.atlassian.net/browse/APIM-1394

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-eojryqhddz.chromatic.com)

### CODE REVIEW

File: helm/values.yaml [hunk 1]
```diff
@@ -168,13 +168,13 @@ smtp:
...
-    username: ${email.username}
-    password: ${email.password}
-    # starttlsEnabled: false
+    host: ${smtp.host}
+    port: ${smtp.port}
+    from: ${smtp.from}
+    subject: ${smtp.subject}
+    username: ${smtp.username}
+    password: ${smtp.password}
+    # starttlsEnabled: ${starttls.properties.enable}`
```
Comment (leleueri) [inline helm/values.yaml]: is it a new value? If so, should we document it and let it start by smtp like the others
(smtp.starttls.properties...)

Comment (sebdevaux) [reply helm/values.yaml]: Did we introduce a breaking with this change?

Comment (kamiiiel) [reply helm/values.yaml]: @leleueri I updated that line. The value was there before but using it by default for "notifiers.smtp" is a new thing.

Comment (kamiiiel) [reply helm/values.yaml]: @sebdevaux the value came from "${email.host}" before but I couldn't find "email.host" anywhere in the values. Did I missed something?

File: helm/values.yaml [hunk 2]
```diff
@@ -168,13 +168,13 @@ smtp:
...
-    username: ${email.username}
-    password: ${email.password}
-    # starttlsEnabled: false
+    host: ${smtp.host}
+    port: ${smtp.port}
+    from: ${smtp.from}
+    subject: ${smtp.subject}
+    username: ${smtp.username}
+    password: ${smtp.password}
+    # starttlsEnabled: ${smtp.properties.starttls.enable}`
```
Comment (sebdevaux) [inline helm/values.yaml]: typo at the end -> `

File: helm/values.yaml (hunk 1 above)
Comment (kamiiiel) [reply helm/values.yaml]: Any thoughts @sebdevaux?

**Comment by sebdevaux** [comment]
After discussing it with @brasseld , we believe this change may be a breaking change. We will make this change for version 4.x and not for 3.x

**Pull request closed without merging**